game/image.hh

```diff
@@ -26,9 +27,10 @@ struct Bitmap {
 	pix::Format fmt;
 	bool linearPremul;  // Is the data linear RGB and premultiplied (as opposed to sRGB and non-premultiplied)
 	bool bottomFirst;  // Upside-down (only used for taking screenshots)
-	Bitmap(unsigned char* ptr = nullptr): ptr(ptr), width(), height(), ar(), timestamp(), fmt(pix::CHAR_RGBA), linearPremul(), bottomFirst() {}
+	Bitmap(unsigned char* ptr = nullptr): ptr(ptr), width(), height(), ar(), timestamp(), fmt(pix::Format::CHAR_RGBA), linearPremul(), bottomFirst() {}
 	void resize(unsigned w, unsigned h) {
-		if (!ptr) buf.resize(w * h * 4); else buf.clear();
+		auto dim = w * h * 4;
+		if (!ptr) buf.resize(dim); else buf.clear();
```

Comment:
Here it's two unsigned ints and vector::resize wants a size_t which, for all intents and purposes, is going to be an unsigned int (although I guess we could change the width and height to be size_t too if that makes you more comfortable)